include/grpcpp/channel_impl.h

```diff
@@ -100,8 +100,9 @@ class Channel final : public ::grpc::ChannelInterface,
   ::grpc::CompletionQueue* CallbackCQ() override;
 
   ::grpc::internal::Call CreateCallInternal(
-      const ::grpc::internal::RpcMethod& method, ::grpc::ClientContext* context,
-      ::grpc::CompletionQueue* cq, size_t interceptor_pos) override;
+      const ::grpc::internal::RpcMethod& method,
+      ::grpc_impl::ClientContext* context, ::grpc::CompletionQueue* cq,
```

Comment:
Do we also need to use grpc_impl for the Completion Queue?

Reply:
Seems OK for now.. didn't give me any issues. We can move it over when necessary..